bootstrap/cmd/kfctl/cmd/apply.go

```diff
@@ -43,8 +43,10 @@ var applyCmd = &cobra.Command{
 			return
 		}
 		options := map[string]interface{}{
-			string(kftypes.OAUTH_ID):     applyCfg.GetString(string(kftypes.OAUTH_ID)),
-			string(kftypes.OAUTH_SECRET): applyCfg.GetString(string(kftypes.OAUTH_SECRET)),
+			string(kftypes.OAUTH_ID):            applyCfg.GetString(string(kftypes.OAUTH_ID)),
```

Comment:
It's fine for now, but this should probably not be passed at apply.

Reply:
thought twice, make them init params and write to app.yaml and encrypt password.